src/Akeneo/Platform/Installer/back/src/Infrastructure/UserConfigurationResetter/UserCatalogLocaleResetter.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Akeneo PIM Enterprise Edition.
+ *
+ * (c) 2022 Akeneo SAS (https://www.akeneo.com)
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Akeneo\Platform\Installer\Infrastructure\UserConfigurationResetter;
+
+use Akeneo\Platform\Installer\Domain\Service\UserConfigurationResetterInterface;
+use Doctrine\DBAL\Connection;
+
+class UserCatalogLocaleResetter implements UserConfigurationResetterInterface
+{
+    public function __construct(private readonly Connection $connection)
```

Comment:
I'd use a const for better readability, wdyt? (same on channel and category)
```suggestion
    private const DEFAULT_LOCALE_CODE = 'en_US';

    public function __construct(private readonly Connection $connection)
```